code/modules/vehicles/cars/civcar.dm

```diff
@@ -0,0 +1,112 @@
+
+/obj/vehicle/sealed/car/civ
+	name = "car"
+	desc = "This used to be a pedestrian-focused station, and now all these cars are ruining it!" //ok boomer
+	icon_state = "car"
+	max_integrity = 300
+	armor = list("melee" = 70, "bullet" = 40, "laser" = 40, "energy" = 0, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
```

Comment:
these are extremely tanky cars

Reply:
goof have you ever tried engaging a car in melee combat

Reply:
according to fallout 3, 4, and 76 this immediately results in a massive explosion